int32 joint_index
float32 torque
